### PR TITLE
Enhancement bottomsheet style

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

File: test/modules/stackables/components/BottomSheet.test.js [hunk 1]
```diff
@@ -57,4 +70,36 @@ describe('BottomSheet', () => {
...
+		});
+
+		it('layouts for portrait and open Menu', async () => {
+			const element = await setup('FooBar', { mainMenu: { open: true }, media: { portrait: true } });
+			const contentElement = element.shadowRoot.querySelector('.bottom-sheet');
+
+			expect(contentElement.innerText).toContain('FooBar');
+			expect(element.shadowRoot.querySelectorAll(`[${TEST_ID_ATTRIBUTE_NAME}]`)).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.bottom-sheet.is-open')).toHaveSize(0);
+		});
```
Comment (taulinger) [inline test/modules/stackables/components/BottomSheet.test.js:101]: Is there no test for `portrait and closed Menu'` needed?

Should we also add tests for changing orientation or opening/closing the menu after initial rendering?

**Review by taulinger — COMMENTED** [review]
Nice, thx!

File: test/modules/stackables/components/BottomSheet.test.js (hunk 1 above)
Comment (alsturm) [reply test/modules/stackables/components/BottomSheet.test.js:102]: done

**Review by taulinger — APPROVED** [review]
Well done!

**Pull request merged**